src/Components/Bio/Bio.tsx

```diff
@@ -6,20 +6,19 @@ import BioEducationAndExperience from "./BioEducationAndExperience";
 import BioPersonalInfo from "./BioPersonalInfo";
 
 const Container = styled.div`
-  box-sizing: border-box;
-  height: 110px;
-  display: table-row-group;
+  width: 1430px;
+  height: auto;
+  display: flex;
   background-color: var(--secondaryDark);
 `;
 
 const BioInfo = styled.div`
   width: 15%;
-  float: left;
+  margin: 25px;
 `;
 
 const Preferences = styled.div`
   width: 85%;
-  float: right;
 `;
 
 export default function Bio() {
```

Comment:
Instead, you might want to encode the values ​​as useState. There will be less work on refactoring when connecting to the backend.